src/Appwrite/Utopia/Response/Model/Target.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class Target extends Model
+{
+    /**
+     * @var bool
+     */
+    protected bool $public = false;
+
+    public function __construct()
+    {
+        $this
+            ->addRule('$id', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Target ID.',
+                'default' => '',
+                'example' => '259125845563242502',
+            ])
+            ->addRule('userId', [
+                'type' => self::TYPE_STRING,
+                'description' => 'User ID.',
+                'default' => '',
+                'example' => '259125845563242502',
+            ])
+            ->addRule('providerId', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Provider ID.',
+                'required' => false,
+                'default' => '',
+                'example' => '259125845563242502',
+            ])
+            ->addRule('providerType', [
+                'type' => self::TYPE_STRING,
+                'description' => 'The type of provider supported by this target.',
+                'default' => '',
+                'example' => 'sms',
+            ])
```

Comment:
Let's remove this since we have removed the collection attribute

Reply:
done this